Fix compilation with builtin_glslang=false

Ref https://github.com/godotengine/godot/pull/93478
Ref https://github.com/godotengine/godot/pull/93471

Patch-Source: https://gitlab.alpinelinux.org/alpine/aports/-/blob/8dd3d8861eff2b4b5171d861e9cde50a92adb93b/testing/godot/glslang.patch

--- a/platform/linuxbsd/detect.py
+++ b/platform/linuxbsd/detect.py
@@ -480,7 +480,7 @@ def configure(env: "SConsEnvironment"):
             env.ParseConfig("pkg-config vulkan --cflags --libs")
         if not env["builtin_glslang"]:
             # No pkgconfig file so far, hardcode expected lib name.
-            env.Append(LIBS=["glslang", "SPIRV"])
+            env.Append(LIBS=["glslang", "SPIRV", "glslang-default-resource-limits"])
 
     if env["opengl3"]:
         env.Append(CPPDEFINES=["GLES3_ENABLED"])
